crates/fuel-core/src/schema/balance.rs

```diff
@@ -1,10 +1,10 @@
 use crate::{
-    database::resource::{
```

Comment:
We should make the whole schema module a sub-module of the new graphql service module.

Reply:
Yep, of course, we do it step by step)